### PR TITLE
6351-confirm-note-totals

### PR DESCRIPTION
**Pull request by bthaile**
confirm form styling to meet mocks and added note with REP and ETH amounts.

This is only going to be for initial reporter when market is in OPEN_REPORTING. I added notes to CH task.

### CODE REVIEW

**Comment by sharkcrayon** [comment]
Related CH story: https://app.clubhouse.io/augur/story/6351/reporting-form-open-reporting-add-confirm-page-note-text

File: src/modules/reporting/components/reporting-report-confirm/reporting-report-confirm.jsx [hunk 1]
```diff
@@ -28,11 +28,16 @@ const ReportingReportConfirm = p => (
             </li>
             <li>
               <span>Gas</span>
-              <span>0.0023 ETH (2.8%)</span>
+              <span>-- ETH</span>
```
Comment (sharkcrayon) [inline src/modules/reporting/components/reporting-report-confirm/reporting-report-confirm.jsx]: Why did you change this to `-- ETH`? Just to show that it's not hooked up yet?

**Comment by sharkcrayon** [comment]
@bthaile How do I view the Reporting form for a market?

File: src/modules/reporting/components/reporting-report-confirm/reporting-report-confirm.jsx [hunk 2]
```diff
@@ -42,6 +47,9 @@ ReportingReportConfirm.propTypes = {
   selectedOutcome: PropTypes.string.isRequired,
   stake: PropTypes.string.isRequired,
   isMarketValid: PropTypes.bool,
+  designatedReportNoShowReputationBond: PropTypes.string,
```
Comment (stephensprinkle) [inline src/modules/reporting/components/reporting-report-confirm/reporting-report-confirm.jsx]: Is this the correct type?

Same for `reporterGasCost`.

File: src/modules/reporting/components/reporting-report-confirm/reporting-report-confirm.jsx [hunk 3]
```diff
@@ -42,6 +47,9 @@ ReportingReportConfirm.propTypes = {
...
+  reporterGasCost: PropTypes.string,
+  isOpenReporting: PropTypes.bool.isRequired,
```
Comment (stephensprinkle) [inline src/modules/reporting/components/reporting-report-confirm/reporting-report-confirm.jsx]: Make sure all `isRequired` properties go before optional.

File: src/modules/reporting/components/reporting-report-confirm/reporting-report-confirm.jsx (hunk 1 above)
Comment (bthaile) [reply src/modules/reporting/components/reporting-report-confirm/reporting-report-confirm.jsx]: yes, to show it's not hooked up.

File: src/modules/reporting/components/reporting-report-confirm/reporting-report-confirm.jsx (hunk 2 above)
Comment (bthaile) [reply src/modules/reporting/components/reporting-report-confirm/reporting-report-confirm.jsx]: It's the same as reporter gas cost, It's actually a formatted number object. I'll make the change to object.

**Comment by bthaile** [comment]
@sharkcrayon need the `Report` button navigation PR merged in. Or can change reporting-view.jsx to nav dispute to reporting-report.jsx and then add `?id=<marketid>`
